src/stdlib/CMakeLists.txt

```diff
@@ -22,6 +22,14 @@ target_include_directories(
         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
 )
 
+target_compile_options(
+        stdlib PRIVATE -Wall -Wextra -Werror
+
+        # TODO: Remove this. We have some current issues with %lld (long long int vs int64_t) that needs to be
+        # TODO: resolved for this. The problem is IIRC that Linux and macOS treated this differently.
+        -Wno-error=format
+)
+
```

Comment:
Deferring the `-Werror=format` issue for later. For now, enabling `-Wall` and `-Werror` is the important part, since without it you can do crazy things like _forgetting to return values_ from a non-`void` function.. and the compiler will happily try to compile your program anyway. :woozy_face: